### PR TITLE
Brand update, Issue #1490

### PR DESCRIPTION
**Pull request by imranadamjee**
* As per the issue, updated to NI in the README's.
* GitHub wiki was good from the start.
* Updated the copyright to the recommendation from ni-central.

 [ x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~~- [x ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

~~- [] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
Updated License and changed all "National Instruments" to "NI" as long as it did not affect code functionality. 

### List issues fixed by this Pull Request below, if any.
Resolves Issue #1490 

### What testing has been done?
Tests pass on local machine, no additional tests needed to be written.

### CODE REVIEW

**Review by marcoskirsch — CHANGES_REQUESTED** [review]
You hand edited the generated files, but not the templates.
You need to modify the templates, run codegeneration, then commit that for the PR.
Thanks!

**Comment by sbethur** [comment]
ok to test

**Comment by sbethur** [comment]
Update the description and reference the relevant GitHub issue

**Comment by sbethur** [comment]
test this please

File: LICENSE [hunk 1]
```diff
@@ -1,8 +1,8 @@
 This is the MIT license: http://www.opensource.org/licenses/mit-license.php
 
-Copyright (c) 2017-2020, National Instruments Corp.
+© 2021 National Instruments Corporation. All rights reserved
```
Comment (sbethur) [inline LICENSE:3]: Follow the format prescribed in https://opensource.org/licenses/mit-license.php, with `National Instruments Corporation` as the copyright holder.

File: LICENSE [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 NI-Digital Pattern Driver, NI-DMM, NI-DCPower, NI-FGEN, NI-SCOPE, NI-SWITCH, NI Switch Executive,
-NI-ModInst, NI-TClk are trademarks of National Instruments.
+NI-ModInst, NI-TClk are trademarks of NI.
```
Comment (sbethur) [inline LICENSE:5]: National Instruments Corporation

File: build/templates/conf.py.mako [hunk 3]
```diff
@@ -63,8 +63,8 @@ master_doc = 'index'
 
 # General information about the project.
 project = 'NI Modular Instruments Python API'
-copyright = '2017-${current_year}, National Instruments'
-author = 'National Instruments'
+copyright = '2017-${current_year}, NI'
```
Comment (sbethur) [inline build/templates/conf.py.mako:66]: National Instruments Corporation

File: src/niswitch/metadata/config.py [hunk 4]
```diff
@@ -43,6 +43,6 @@
             'python_name': 'channels'
         }
     ],
-    'session_class_description': 'An NI-SWITCH session to a National Instruments Switch Module',
+    'session_class_description': 'An NI-SWITCH session to a NI Switch Module',
```
Comment (sbethur) [inline src/niswitch/metadata/config.py:46]:
```suggestion
    'session_class_description': 'An NI-SWITCH session to an NI Switch Module',
```

Applies to other places too.

File: src/nidigital/metadata/attributes.py [hunk 5]
```diff
@@ -132,7 +132,7 @@
         'access': 'read only',
         'channel_based': False,
         'documentation': {
-            'description': 'Returns a string ("National Instruments") that contains the name of the manufacturer of the digital pattern instrument.\n'
+            'description': 'Returns a string ("NI") that contains the name of the manufacturer of the digital pattern instrument.\n'
```
Comment (sbethur) [inline src/nidigital/metadata/attributes.py:135]: Revert this change. The software still returns the expanded name.

File: src/nidigital/metadata/attributes.py [hunk 6]
```diff
@@ -150,7 +150,7 @@
         'access': 'read only',
         'channel_based': False,
         'documentation': {
-            'description': 'Returns a string ("National Instruments") that contains the name of the vendor that supplies the NI-Digital Pattern Driver.\n'
+            'description': 'Returns a string ("NI") that contains the name of the vendor that supplies the NI-Digital Pattern Driver.\n'
```
Comment (sbethur) [inline src/nidigital/metadata/attributes.py:153]: Revert this change. The software still returns the expanded name.

File: src/nidcpower/examples/nidcpower_advanced_sequence.py [hunk 7]
```diff
@@ -50,7 +50,7 @@ def example(resource_name, options, voltage_max, current_max, points_per_output_
 
 def _main(argsv):
     parser = argparse.ArgumentParser(description='Output ramping voltage to voltage max, then ramping current to current max.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument('-n', '--resource-name', default='PXI1Slot2/0, PXI1Slot3/0-1', help='Resource name of National Instruments SMUs')
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2', help='Resource name of a NI SMU')
```
Comment (sbethur) [inline src/nidcpower/examples/nidcpower_advanced_sequence.py:53]: Why did the default change? Looks like a merge issue.

**Review by sbethur — CHANGES_REQUESTED** [review]

**Comment by sbethur** [comment]
The changes here were submitted by #1714 and #1734.

**Pull request closed without merging**